src/lib.rs

```diff
@@ -1,11 +1,18 @@
 use std::error::Error;
+// config
+pub mod config;
+pub use crate::config::Config;
 
 // CLI input output
 mod cli;
 
 // game core
 
-fn run_game_loop(config: &mut Config, guess: char, player_state: &mut PlayerState) -> GameState {
+fn run_game_loop(
+    config: &mut config::Config,
```

Comment:
Could the `config` be an immutable here as well?

Reply:
yeah great catch